### PR TITLE
Capture higher log level output on oc failures

### PR DESCRIPTION
**Pull request by stevekuznetsov**
Signed-off-by: Steve Kuznetsov <skuznets@redhat.com>

/cc @smarterclayton 
/assign @petr-muller @bbguimaraes

### CODE REVIEW

File: hack/monitor_prow.py [hunk 1]
```diff
@@ -17,16 +17,26 @@
...
+def run_oc(args):
+    command = ['oc', '--loglevel', '10', '--namespace', 'ci']
+    command.extend(args)
+    process = subprocess.Popen(command, stdout=subprocess.PIPE, stderr=subprocess.PIPE)
+    stdout, stderr = process.communicate()
+    if process.returncode != 0:
+        print(stderr.decode('utf-8'))
+        raise subprocess.CalledProcessError(returncode=process.returncode, cmd=command, output=stdout)
+
+    return stdout.decode('utf-8')
```
Comment (bbguimaraes) [inline hack/monitor_prow.py]: What's wrong with little `exec_cmd` over there?  ^

Comment (stevekuznetsov) [reply hack/monitor_prow.py]: doesn't grab stderr in a separate thign

Comment (stevekuznetsov) [reply hack/monitor_prow.py]: (which is how you get loglevel output from not garbling your JSON etc)

File: hack/monitor_prow.py [hunk 2]
```diff
@@ -17,16 +17,26 @@
 RESET = exec_cmd('tput', 'sgr0')
 CLEAR = exec_cmd('tput', 'clear')
 
+
+def run_oc(args):
+    command = ['oc', '--loglevel', '10', '--namespace', 'ci']
+    command.extend(args)
```
Comment (petr-muller) [inline hack/monitor_prow.py]: This could have been `command = [...] + args`

**Comment by stevekuznetsov** [comment]
holy shit i didnt break `pylint`?

File: hack/monitor_prow.py [hunk 3]
```diff
@@ -17,16 +17,26 @@
...
+    process = subprocess.Popen(command, stdout=subprocess.PIPE, stderr=subprocess.PIPE)
```
Comment (petr-muller) [inline hack/monitor_prow.py]: Any reason why not to use `subprocess.run` ([subprocess.run](https://docs.python.org/3/library/subprocess.html#subprocess.run))?  Would end up being a little bit pythonic:

```
try:
  process = subprocess.run(command, capture_output=True, check=True)
except subprocess.CalledProcessError as exc:
  print(exc.stderr.decode('utf-8')
  raise

return process.stdout.decode('utf-8')
```

File: hack/monitor_prow.py (hunk 1 above)
Comment (bbguimaraes) [reply hack/monitor_prow.py]: Does `oc logs` combine `stdout` and `stderr` into its `stdout`?  Or will the `--loglevel 10` output get mixed with the actual logs?

**Comment by stevekuznetsov** [comment]
@petr-muller updated

File: hack/monitor_prow.py (hunk 1 above)
Comment (stevekuznetsov) [reply hack/monitor_prow.py]: All logger output is `stderr`

**Pull request merged**